lib/src/web/rtc_peerconnection_impl.dart

```diff
@@ -265,6 +275,9 @@ class RTCPeerConnectionWeb extends RTCPeerConnection {
   @override
   Future<RTCRtpSender> addTrack(MediaStreamTrack track,
       [List<MediaStream> streams]) {
+    var _track = track as MediaStreamTrackWeb;
+    var _stream = streams[0] as MediaStreamWeb;
+    _jsPc.addTrack(_track.jsTrack, _stream.jsStream);
     // TODO: implement addTrack
     throw UnimplementedError();
```

Comment:
This method is half implemented.
And probably, we need to add the track to all the streams,  Maybe a for a for loop ? 
@rostopira can you comment on this?

Reply:
Regarding the js part, I wonder if there is a better way to implement it.
like https://pub.dev/packages/js.

Reply:
Another idea is to use js/js.dart to package directly from the browser environment, but I am not sure that it is effective.

like this: https://pub.dev/packages/import_js_library.

Reply:
Yeah, Perhaps it is the best choice at this stage.